Source/RxCocoa/UIScrollView+reachedBottom.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  UIScrollView+reachedBottom.swift
+//  RxSwiftExt
+//
+//  Created by Anton Nazarov on 09/05/2019.
+//  Copyright © 2019 RxSwift Community. All rights reserved.
+//
+
+import UIKit
+import RxSwift
+import RxCocoa
+
+public extension Reactive where Base: UIScrollView {
+    /**
+     Shows if the the bottom of the UIScrollView is reached.
+     - parameter offset: A threshhold indicating the bottom.
+     - returns: Sequence that emits when the bottom of the base UIScrollView is reached.
+     */
+    func reachedBottom(offset: CGFloat = 0.0) -> Signal<Void> {
+        return contentOffset.filterMap { contentOffset in
```

Comment:
I don't see the need for `filterMap`. Why not `map`?

Reply:
Because we should filter events in case contentOffset is not near bottom